src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2620,6 +2630,19 @@ public function getAttribute($key)
     }
 
     /**
+     * Retrieve a value (raw) for an attribute from this model.
+     *
+     * @param string $key
```

Comment:
Okay no problem - so your request still isn't very clear. Where do you want 2 spaces?